log/test/logtest.go

```diff
@@ -1,19 +1,34 @@
-// Copyright 2018-2019 Celer Network
+// Copyright 2018-2020 Celer Network
 
 // Try run with different flags. Example:
-// go run clog/test/clogtest.go -logcolor -loglevel=debug
-// go run clog/test/clogtest.go -loglevel=warn -loglocaltime -loglongfile
+// go run logtest.go -logcolor -loglevel=debug
+// go run logtest.go -loglevel=warn -loglocaltime -loglongfile
+// go run logtest.go -testoutput
 
 package main
 
 import (
 	"flag"
+	"fmt"
 
 	"github.com/celer-network/goutils/log"
 )
 
+type TestOutput struct {
+}
+
+func (cb TestOutput) Write(output []byte) (n int, err error) {
```

Comment:
nit: it's a nicer Go pattern to use a pointer for the object receiver instead of a struct
in case somebody copy/pastes this code